admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -745,7 +761,10 @@ var form = (function() {
 
   function switchLanguage(iso_code) {
     $('div.translations.tabbable > div > div.translation-field:not(.translation-label-' + iso_code + ')').removeClass('show active');
-    $('div.translations.tabbable > div > div.translation-field.translation-label-' + iso_code).addClass('show active');
+
+    const langueTabSelector = 'div.translations.tabbable > div > div.translation-field.translation-label-' + iso_code;
```

Comment:
```suggestion
    const langueTabSelector = `div.translations.tabbable > div > div.translation-field.translation-label-${iso_code}`;
```

Reply:
I wish I could but this is an ES6 feature right? this file is from the default name which is not transpiled, so I'm worried about the compatibility with old browsers

Reply:
Oh, you're right, it is the old theme:(